lib/Elastica/JSON.php

```diff
@@ -28,6 +28,17 @@ public static function parse(/* inherit from json_decode */)
             $args[] = true;
         }
 
+        // add code to set use JSON_BIGINT_AS_STRING by default
+        if (!isset($args[2])) {
```

Comment:
We should mention here why this code is here by linking to the issue: https://github.com/ruflin/Elastica/issues/717